sysl2/sysl/swagger/endpoints.go

```diff
@@ -29,12 +36,45 @@ func initEndpoint(path string,
 	res := Endpoint{
 		Path:        path,
 		Description: op.Description,
-		Responses:   op.Responses,
+		Responses:   buildResponses(path, op.Responses, types, logger),
 		Params:      buildParameters(op.Parameters, types, globals, apiParams, logger),
 	}
 	return res
 }
 
+func buildResponses(path string, responses *spec.Responses, types TypeList, logger *logrus.Logger) []Response {
+	var outs []Response
+
+	fn := func(n string) Response {
+		return Response{Text: n}
+	}
+
+	for statusCode, response := range responses.StatusCodeResponses {
+		if schema := response.Schema; schema != nil {
+
+			t, found := types.FindFromSchema(*schema, &typeData{logger: logger})
+			if !found {
+				logger.Panicf("Responses type for code %d not found, endpoint: %s", statusCode, path)
```

Comment:
Does this represent a program bug or bad input?

Reply:
bad input

Reply:
OK, will do